### ISSUE TITLE
How best to exclude dot files?

### ISSUE THREAD

**Issue by ghost** [braceio/tags#8]
If I wanted to exclude both "_file" and ".file" what would the exclude pattern look like? It's currently including all the .git files as I'm running this at the root of the repo.

**Comment by colevscode** [comment]
Yeah, there needs to be a conf file that lets you specify multiple excludes and multiple includes.
